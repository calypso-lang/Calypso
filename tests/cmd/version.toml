bin.name = "calypso"
args = "--version"
