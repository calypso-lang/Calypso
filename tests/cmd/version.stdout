calypso 0.1.0
